### PR TITLE
STUFF String Function: #13

### PR DESCRIPTION
**Pull request by Lilit0x**
Implemented STUFF for string functions

### CODE REVIEW

File: crates/gitql-ast/src/function.rs [hunk 1]
```diff
@@ -242,6 +250,26 @@ fn text_char(inputs: Vec<Value>) -> Value {
...
+        return Value::Text(text);
+    }
+    if start as usize > text.len() || length as usize > text.len() {
+        return Value::Text(text);
+    }
+
+    let mut text = text.chars().collect::<Vec<_>>();
+    let new_string = new_string.chars().collect::<Vec<_>>();
+    text.splice(start as usize..(start + length), new_string);
+
```
Comment (AmrDeveloper) [inline crates/gitql-ast/src/function.rs:269]: Looks good for me but we need to double check if std has function for this purpose

Comment (Lilit0x) [reply crates/gitql-ast/src/function.rs:269]: I don't think so, I checked before implementing it

**Review by AmrDeveloper — APPROVED** [review]

**Comment by AmrDeveloper** [comment]
Good work @Lilit0x, please resolve the conflicts

**Comment by Lilit0x** [comment]
@AmrDeveloper , I keep trying to squash the commits, but there's still a conflict after I push from my local back here.

**Comment by AmrDeveloper** [comment]
@Lilit0x Check Conflicts to remove related code to substring function

**Pull request closed without merging**